test/module/irohad/consensus/yac/mocks/crypto_provider.hpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gmock/gmock.h>
+
+#include "consensus/yac/yac_crypto_provider.hpp"
+#include "cryptography/crypto_provider/crypto_defaults.hpp"
+
+#include "module/shared_model/interface_mocks.hpp"
+
+#ifndef IROHA_CRYPTO_PROVIDER_HPP
+#define IROHA_CRYPTO_PROVIDER_HPP
+namespace iroha {
+  namespace consensus {
+    namespace yac {
+
+      /**
+       * Creates test signature with empty signed data, and provided pubkey
+       * @param pub_key - public key to put in the signature
+       * @return new signature
+       */
+      std::shared_ptr<shared_model::interface::Signature> createSig(
```

Comment:
Maybe put this method in some file related to `shared_model` instead? Because it can be reused in other tests.

Reply:
I can't find any useful places for createSig. Could you suggest something?